src/scss/components/_resume-page.scss

```diff
@@ -1,604 +1,212 @@
-    .cmp-body--resume {
-        padding-top: 35%;
+        content: "\003D\003E";
+        font-size: 1rem;
+        position: absolute;
+        left: -2rem;
+        
+        @media (min-width: general.$bp-medium) {
+          top: 0.33rem;
+        }
+      }
     }
+  }
 
-    .hero-section--resume .header .across-top {
-        display: block;
+  &__list {
+    &.border-left {
+      padding-left: 1rem;
     }
+  }
 
-    .hero-section--resume .header .across-top a.icon {
-        margin-left: 85%;
-        margin-bottom: 7.5%;
-    }
+  &__list-item {
+    font-size: 1rem;
 
-    .grid-container {
-        padding-bottom: 50%;
-        margin-bottom: 25%;
-    }
-    
-    .hero-section--resume .header h1 {
-        font-size: 3em;
+    @media (min-width: general.$bp-medium) {
+      font-size: 1.25rem;
     }
 
-    #about {
-        line-height: 140%;
+    &--heading {
+      color: var(--color-bright-coral);
+      font-weight: 600;
     }
+  }
 
-    #about .hello {
-        display: flex;
-        align-items: center;
-        padding-bottom: 8%;
-    }
+  &__sub-list {
+    margin-bottom: 1rem;
 
-    #education h2, 
-    #experience h2, 
-    #skills h2 {
-        padding-bottom: 8%;
+    @media (min-width: general.$bp-medium) {
+        margin-bottom: 2rem;
     }
 
-    #education ul li.year {
-        padding-bottom: 3%;
-        padding-top: 0;
+    & > & {
+      margin-top: 1rem;
     }
+  }
 
-    #education ul li.item {
-        padding-bottom: 5%;
-    }
+  &__sub-list-item {
+    margin-bottom: 0.5rem;
 
-    #education ul li.sub-items {
-        padding-bottom: 5%;
-    }
+    @media (min-width: general.$bp-medium) {
+        font-size: 1rem;
+      }
 
-    #education ul li:nth-child(5), 
-    #education ul li:nth-child(8) {
-        margin-bottom: 5%;
+    &--title {
+      font-size: 1.125rem;
+      font-weight: 600;
+      
+      @media (min-width: general.$bp-medium) {
+          font-size: 1.25rem;
+          margin-bottom: 1rem;
+      }
     }
 
-     #experience .experience-container {
-        padding-bottom: 0;
-    } 
-
-    #experience .experience-container .fraser, 
-    #experience .experience-container .wwt, 
-    #experience .experience-container .marysville {
-        border-left: none;
-        border-bottom: 2px solid #e5e5e5;
-        padding-left: 5%;
-        margin-bottom: 10%;
-    }
+    &--with-marker {
+      position: relative;
+      left: 1.5rem;
+      width: calc(100% - 1.5rem);
 
-    #experience .experience-container .fraser p.date, 
-    #experience .experience-container .wwt p.date,
-    #experience .experience-container .marysville p.date {
-        padding-bottom: 3%;
+      &::before {
+        position: absolute;
+        background: url('../../public/usedresources/arrow-marker.svg') center no-repeat;
+        background-size: contain;
+        content: "";
+        top: 0;
+        left: -1.5rem;
+        width: 1rem;
+        height: 1rem;
```

Comment:
Setting the `height` of the list marker equal to the `line-height` of a list item should align the marker to the center of the text here.

Before:
<img width="591" alt="Screenshot 2024-03-13 at 9 54 11 AM" src="https://github.com/marissahuysentruyt/marissahuysentruyt/assets/53844657/65f344b4-890d-4e96-b5dc-a038c56d25bd">

After:
<img width="593" alt="Screenshot 2024-03-13 at 9 54 26 AM" src="https://github.com/marissahuysentruyt/marissahuysentruyt/assets/53844657/ea266487-058f-4769-8534-2540b7be5a46">

Reply:
great catch- thanks!